src/components/inline-link/inline-link.module.css

```diff
@@ -40,3 +43,19 @@ ones set in the surrounding text.
 		}
 	}
 }
+
+.secondary {
+	color: var(--token-color-foreground-primary);
+
+	&:active {
+		color: var(--token-color-foreground-faint);
+	}
+
+	&:hover {
+		color: var(--token-color-foreground-faint);
```

Comment:
The HDS documentation didn't give clear data on the exact colors used for active states so I used my best judgement. _aside_ They use images instead of live components on the site which is a challenge. Not sure if there's a better place to look for these values.

Reply:
I will confirm

Reply:
Default: `foreground-strong`
Hover: `foreground-primary`
Focus: `foreground-strong` + focus outline
Active: `foreground-faint`

I originally set these at `foreground-primary` so let's update the default then use the HDS hover/focus/active values listed above.

Reply:
Awesome, ty. will update!

Reply:
So just to confirm, you want `foreground-primary` for default? Then what do you want for hover since that's the same?

Reply:
Also, is there a spec or any info on `:visited` state?

Reply:
Sorry all secondary links should be the following by default.

Default: `foreground-strong`
Hover: `foreground-primary`
Focus: `foreground-strong` + focus outline
Active: `foreground-faint`

HDS doesn't have specific styles for :visited but we should stay consistent with what we have for inline links!

Reply:
Updated!